src/autocomplete.py

```diff
@@ -0,0 +1,20 @@
+from trie import Trie
+
+
+class Autocomplete(object):
+    """Create autocomplete class"""
+    def __init__(self, words_list, max_completions=5):
+        """creates words and sets up max of 5 completions."""
+        self.trie = Trie(iterable=words_list)
+        self.max_completions = max_completions
+
+    def __call__(self, item=None):
+        result = []
+        count = 0
+        if item and isinstance(item, type('alpha')):
+            for w in self.trie.traversal(item[0]):
+                if count < self.max_completions:
```

Comment:
You can more efficient than this; you want your loop to stop once you've reached `max_completions`